admittance_controller/include/admittance_controller/admittance_rule_impl.hpp

```diff
@@ -331,16 +331,20 @@ void AdmittanceRule::process_wrench_measurements(
 
 const control_msgs::msg::AdmittanceControllerState & AdmittanceRule::get_controller_state()
 {
+  for (size_t i = 0; i < 6; ++i)
```

Comment:
```suggestion
  for (size_t i = 0; i < NUM_MAX_SUPPORTED_JOINTS; ++i)
```

in the interest of keeping it somewhat readable and maintainable, let's use a named constant instead of a magic number

Reply:
Using a constant is fine, but this 6 isn't the number of max supported joints, it's the number of degrees of freedom for the admittance controller (3 translation + 3 rotation)

Reply:
Oooops! Even more reason for a better name ;)

Reply:
Done